unity-environment/Assets/ML-Agents/Scripts/Brain.cs

```diff
@@ -54,7 +54,7 @@ public class BrainParameters
      * the state
      * <br> If discrete : The number of possible values the state can take*/
 
-    [Range(1, 10)]
+    [Range(1, 50)]
```

Comment:
Is it necessary to go up to 50  ?

Reply:
Lots of environments (like Hallway) could take advantage of getting a long history of observations. No reason to artificially limit it to developers, especially when it may be more robust than LSTM.